PyStemmusScope/bmi/variable_reference.py

```diff
@@ -28,15 +28,43 @@ class BmiVariable:
         keys=["fluxes", "Resp"],
     ),
     BmiVariable(
-        name="evaporation_total",
+        name="precipitation",
+        dtype="float64",
+        input=True,
+        output=False,
+        units="cm s-1",
+        grid=0,
+        keys=["ForcingData", "Precip_msr"],
+        all_timesteps=True,
+    ),
+    BmiVariable(
+        name="applied_infiltration",
         dtype="float64",
         input=False,
         output=True,
         units="cm s-1",
         grid=0,
-        keys=["EVAP"],
+        keys=["ForcingData", "applied_inf"],
         all_timesteps=True,
     ),
+    BmiVariable(
+        name="evaporation_total",
+        dtype="float64",
+        input=False,
+        output=True,
+        units="cm s-1",
+        grid=0,
+        keys=["Evap"],
+    ),
+    BmiVariable(
+        name="transpiration_total",
```

Comment:
I can understand the "evaporation_total" name, as transpiration is separate (and you can have soil evaporation separately as well) but why does transpiration have `_total` in the name?

You cannot get `transpiration_understory` and `transpiration_canopy` right?

Reply:
I actually agree with you, I was trying to give a similar name to `evaporation_total`.  However, I don't get why it is called `evaporation_total`. What STEMMUS-SCOPE calculates is `soil_evaporation` and not total evaporation. Physically, there is still one evaporation component that is missing (called `canopy interception`), but it is not simulated by STEMMUS_SCOPE yet. 

So maybe we should change the names 
`evaporation_total` to `evaporation`
`transpitation_total` to `transpiration`

Reply:
> What STEMMUS-SCOPE calculates is soil_evaporation and not total evaporation. 

Then the evaporation variable should be named `soil_evaporation`!